src/input_box/html/inputbox.html

```diff
@@ -69,6 +55,86 @@
     const KEY_RETURN = 13;
     const KEY_ESC = 27;
 
+    Vue.component('su-textbox', {
+      props: ['index', 'input'],
+      template: `
+        <div>
+          <label
+              class="col-sm-3 control-label"
+              v-bind:for.once="'suCheckbox' + index">{{ input.label }}: </label>
+          <div class="col-sm-9">
+            <input
+                class="form-control"
+                v-bind:id.once="'suCheckbox' + index"
```

Comment:
Reminder that all the component templates have the same "suCheckbox" prefix for the ID.
Do they need to reflect their `input.type` ? ... or is a generic prefix like "su-control-" more appropriate ?

Reply:
`su-control-` is a better pattern, since the index is based on the total number of controls. Thanks for catching that.